### PR TITLE
Adding recommended configuration for 500 node cluster and miscellaneous fixes

### PR DESCRIPTION
**Pull request by divyenpatel**
Here is the summary of change.
 - Updated `Prerequisites` title in `home_sidebar.yml`
 - Updated `faqs.md` and mentioned that VCP is tested with 500 node cluster so far.
 - Added recommended configuration for large cluster.

### CODE REVIEW

File: documentation/faqs.md [hunk 1]
```diff
@@ -2,17 +2,22 @@
 title: FAQs
 ---
 
-## Is vSphere Cloud Provider ready for production?
-vSphere Cloud Provider is still in beta.
-
 ## What is the biggest Kubernetes cluster it has been tested for?
-It has been tested on eight node cluster so far.
+It has been tested on 500 node cluster so far. Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/largescaledeployment.html) for details about recommended configuration.
```
Comment (rjog) [inline documentation/faqs.md]: Will it make more clear with "500 node kubernetes cluster"?
In our environment, we have ESXi cluster, vSAN cluster, cabinets cluster.. too many clusters!

File: documentation/faqs.md [hunk 2]
```diff
@@ -2,17 +2,22 @@
...
 ## Where can I find required Roles and Privileges for the vCenter User for vSphere Cloud Provider?
  Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/existing.html#create-roles-add-privileges-to-roles-and-assign-them-to-the-vsphere-cloud-provider-user-and-vsphere-entities).
 
-## Where can I find the list of vSAN, VMFS and NFS features supported by vSphere Cloud Provider?
-Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html). Please report in case you find any features are missing.
+
+## Which vSphere Datastore types are supported by vSphere Cloud Provider?
+VMFS, NFS, vSAN and VVOl datastore types are supported by vSphere Cloud Provider.
+
+## Where is the details about Storage Policy Based Management (SPBM) for dynamic provisioning of volumes?
```
Comment (rjog) [inline documentation/faqs.md]: Q should be:

"Can SPBM be used with dynamic volume provisioning?"

File: documentation/faqs.md [hunk 3]
```diff
@@ -2,17 +2,22 @@
...
+Please refer to this [section](/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html). 
+Also refer to [Storage Policy Management inside kubernetes](https://github.com/kubernetes/examples/blob/master/staging/volumes/vsphere/README.md#storage-policy-management-inside-kubernetes) for more details.
+Please report in case you find any features are missing.
```
Comment (rjog) [inline documentation/faqs.md]: This should be generically told somewhere else.
May be at the bottom of this page.

File: documentation/largescaledeployment.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+```
+Capacity:
+ cpu:     8
+ memory:  8174428Ki
+```
+**Note:** Here 8 CPU cores means 8000 milli-cores.
+
+Worker nodes are allocated 2 GB and 2 CPUs.
+
+**Note:** CPU/Memory allocation for nodes can be adjusted based on the workload requirement. Above is the minimum recommended allocation.
```
Comment (rjog) [inline documentation/largescaledeployment.md]: Above is the minimum required allocation.

File: documentation/largescaledeployment.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+| Pod | CPU Requests | Memory Requests |
+| ------ | ------ | ------ |
+| apiserver | 1200m (15%) | 1650Mi |
+| controller-manager | 1600m (20%) | 2200Mi |
+| scheduler | 800m (10%) | 1100Mi |
+| etcd | 1200m (15%) | 1650Mi |
+
+**Note:** Here resource `requests` is set on system pods and not the `limit`, so that if Pod requires more CPU/Memory then available CPU/Memory can be utilized during the peak time.
+
+For rest of the Pods on the master node (Proxy, DNS and flannel) we have 40% cpu available.
```
Comment (rjog) [inline documentation/largescaledeployment.md]: (Proxy, DNS and flannel etc.)

**Review by rjog — CHANGES_REQUESTED** [review]

File: documentation/faqs.md (hunk 1 above)
Comment (divyenpatel) [reply documentation/faqs.md]: fixed.

File: documentation/faqs.md (hunk 2 above)
Comment (divyenpatel) [reply documentation/faqs.md]: changed question.

File: documentation/faqs.md (hunk 3 above)
Comment (divyenpatel) [reply documentation/faqs.md]: removed this line.

File: documentation/largescaledeployment.md (hunk 4 above)
Comment (divyenpatel) [reply documentation/largescaledeployment.md]: Fixed.

File: documentation/largescaledeployment.md (hunk 5 above)
Comment (divyenpatel) [reply documentation/largescaledeployment.md]: Fixed.

**Comment by divyenpatel** [comment]
@rohitjogvmw Addressed review comments https://github.com/vmware/vsphere-storage-for-kubernetes/pull/14/commits/f8d30a675c396de6b0b6d0ac4452811d779ef051

**Review by rjog — APPROVED** [review]
lgtm

**Pull request merged**